jwt/api_jwt.py

```diff
@@ -76,7 +76,7 @@ def decode_complete(
         detached_payload: Optional[bytes] = None,
         # passthrough arguments to _validate_claims
         # consider putting in options
-        audience: Optional[str] = None,
+        audience: Optional[Union[str, Iterable]] = None,
```

Comment:
We can also do :
```suggestion
        audience: Optional[Union[str, Iterable[str]]] = None,
```

But as a `str` is a `Iterable[str]` itself,  I don't know what would be the best. 
This version looks already better to me, so (IMHO) we can go this way.